includes/functions/functions_lookups.php

```diff
@@ -676,7 +676,8 @@ function zen_get_products_allow_add_to_cart($lookup) {
 /*
  * Look up SHOW_XXX_INFO switch for product ID and product type
  */
-    function zen_get_show_product_switch_name($lookup, $field, $suffix= 'SHOW_', $prefix= '_INFO', $field_prefix= '_', $field_suffix='') {
+    function zen_get_show_product_switch_name($lookup, $field, $prefix= 'SHOW_', $suffix= '_INFO', $field_prefix= '_', 
+                                              $field_suffix='') {
```

Comment:
Other than adding a line-break to wrap to the next line in 4 places in this file, was there anything else changed in this file?